inclusive-design/wecount.inclusivedesign.ca
# test: use a netlify function to test if it can create a remote branch

## Description

Test if a Netlify function can use its temp directory to clone a remote repo and push up a new remote branch. The documentation about the execution of a Lambda function has 512M space in the /tmp directory can be found at [here](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-context.html#runtimes-lifecycle-shutdown).

This PR is a continuation of https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/445 that failed at cloning a remote repo probably because it was not using `/tmp` to save the clone locally.

The deploy of this pull request requires an environment variable `GITHUB_ACCOUNT_URL` set to `https://{username}:{personal-access-token}@github.com/{username}/{repo-name}.git`

## Steps to test

1. Send a POST request to "/api/createBranch" endpoint, for example: `curl -X POST [host-server]/api/createBranch`
2. Check the response. 
3. If the response shows a failure, check the netlify log;

**Expected behavior:** 

The remote branch "from-netlify" should be created on "cindyli/inverted-wordles" repo.
